### ISSUE TITLE
[Bug]fix bug about softmax

### ISSUE THREAD

**Issue by rachelsunrh** [Angel-ML/PyTorch-On-Angel#63]
fix bug about softmax
